src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const tableNames = document.querySelectorAll('thead th');
+
+tableNames.forEach((element, index) => {
+  element.addEventListener('click', () => sortTable(index));
+});
+
+function sortTable(index) {
+  const tableBody = document.querySelector('tbody');
+  const rows = Array.from(tableBody.querySelectorAll('tr'));
+
+  rows.sort((rowA, rowB) => {
+    const cellA = rowA.cells[index].textContent.trim();
+    const cellB = rowB.cells[index].textContent.trim();
+
+    return compareValues(cellA, cellB, index);
+  });
+
+  tableBody.append(...rows);
+}
+
+function compareValues(cellA, cellB, index) {
+  if (index <= 1) {
```

Comment:
The condition `index <= 1` assumes that the first two columns are always text. Ensure this condition aligns with the actual table structure and requirements.